ui/app/AppLayouts/Wallet/popups/swap/SwapSignApprovePopup.qml

```diff
@@ -0,0 +1,267 @@
+import QtQuick 2.15
+import QtQuick.Layouts 1.15
+import QtQml.Models 2.15
+
+import StatusQ 0.1
+import StatusQ.Core 0.1
+import StatusQ.Popups.Dialog 0.1
+import StatusQ.Controls 0.1
+import StatusQ.Components 0.1
+import StatusQ.Core.Theme 0.1
+import StatusQ.Core.Utils 0.1 as SQUtils
+
+import shared.controls 1.0
+
+import AppLayouts.Wallet.controls 1.0
+
+import utils 1.0
+
+// TODO: this is only temporary placeholder and should be replaced completely by https://github.com/status-im/status-desktop/issues/14785
+StatusDialog {
+    id: root
+
+    required property bool loading
+    required property SwapSignApproveInputForm swapSignApproveInputForm
+    required property SwapSignApproveAdaptor adaptor
+
+    signal sign()
+    signal reject()
+
+    objectName: "swapSignApproveModal"
+    implicitWidth: 480
+    padding: 20
+    title: qsTr("Approve spending cap")
```

Comment:
A bit confused; is this "Approve spending cap" popup as the title suggests, or "Sign/Approve" as the filename does?

Reply:
I will add a condition for sign in my next PR.